docker/build-from-source.sh

```diff
@@ -36,7 +36,7 @@ make install
 cd /_src/pinba2
 ./buildconf.sh
 ./configure --prefix=/_install/pinba2 \
-	--with-mysql=/usr \
+	--with-mysql=/home/builder/rpm/mariadb-10.1.26 \
```

Comment:
@mkevac this will fail with next release of MariaDB. We should either get the directory name dynamically or explicitly set versions in `Dockerfile`.